src/extension.js

```diff
@@ -74,7 +75,7 @@ function activate(context) {
         case 'shoot':
           vscode.window
             .showSaveDialog({
-              defaultUri: lastUsedImageUri,
+              defaultUri: getFileName(),
```

Comment:
* Call to the `getFileName` function instead of using the global variable as it becomes a derived value.